ObjectCreation/boxcreator/boxcreator.py

```diff
@@ -1,157 +1,178 @@
 
 import FreeCAD
```

Comment:
Could you please use `import FreeCAD as app` and replace `FreeCAD` with `app` in the code? This is to be consistent with how the rest of the repository should import `FreeCAD` (WIP).

Reply:
done